entrypoint.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env sh
 
-set -o errexit
-
-IDEA_JDK="/app/idea-IC/jre64"
-export IDEA_JDK
-TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
-export TMPDIR
+export JAVA_HOME=/app/jdk
+export JAVA_TOOL_OPTIONS=-Djava.io.tmpdir=${XDG_CACHE_HOME}/tmp
+export MAVEN_HOME=/usr/lib/sdk/openjdk11/maven
```

Comment:
For me, this is preventing all these tools (Ant, Gradle, Maven) to store their assets in the default location they use: `~/.m2` and `~/.gradle` (I didn't test Ant). I think they might be using some path within the container, and that's going to evaporate once the application is closed.

I think users wouldn't expect that. Moreover, sometimes there are settings that needs to be applied globally, and that's usually configured inside `~/.m2` and/or `~/.gradle` (again, I can't talk for Ant, but I would assume it's the same story).

Reply:
I don't have any issue on ~/.m2 folder. It's saving there artifacts from the embedded maven build on the right panel and from the terminal. They both work fine and the artifacts are persisted across restarts. Did you try the latest build? Now the flatpak has access to those directories.